package.json

```diff
@@ -20,10 +27,11 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook",
     "pre-commit": "lint-staged",
-    "build-storybook": "build-storybook -c .storybook -o .out"
+    "build-storybook": "build-storybook -c .storybook -o .out",
+    "eslint": "eslint . --ext .tsx,.ts"
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,css, scss}": [
+    "src/**/*.{css, scss, ts, tsx}": [
```

Comment:
You can't eslint css files.

Reply:
You changed it to style lint in other PR so it's fine.